scripts/rename-mjs.sh

```diff
@@ -0,0 +1,4 @@
+for file in ./build/module/**/*.js; do
```

Comment:
Is there a way to export to those suffix from tsconfig?

Reply:
Checked with cloud and this is his idea
```
// tsconfig.json
{
  "compilerOptions": {
    "module": "ES2020",
    "moduleResolution": "node",
    "outDir": "./dist",
    "target": "ES2020"
  }
}
```

Reply:
This option does not exist yet in tsconfig unfortunately, it's always .js.

Only when you use a bundler like tsup you can output both with one build command.